### DISCUSSION TITLE
Chat memory not working with "Conversational Retriever QA Chain"

### DISCUSSION THREAD

**Discussion by room3dev** [FlowiseAI/Flowise#685] [Q&A]
It working for "Conversation Chain" but not with "Conversational Retriever QA Chain" (Both for Web & API, flowise@1.3.0)

Conversation Chain (Working):
![image](https://github.com/FlowiseAI/Flowise/assets/25867378/1f0f2b8c-1f64-48b7-8a81-2afa305c9b65)

Conversational Retriever QA Chain (Not working):
![image](https://github.com/FlowiseAI/Flowise/assets/25867378/242264e9-89c7-4e20-8a36-09a75a0c3398)
![image](https://github.com/FlowiseAI/Flowise/assets/25867378/1d889e79-aa73-4a0f-8e5f-0611c3563085)

Not sure If this a bug or I did something wrong.

**Comment by chungyau97** [comment]
#687
